project/plugins.sbt

```diff
@@ -1,9 +1,5 @@
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.5")
+addSbtPlugin("com.github.sbt" % "sbt-release" % "1.1.0")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
-
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "0.2.2")
-
-
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
built in to SBT now

Reply:
I think you have to replace it with `addDependencyTreePlugin` though